cursorless.nvim/README.md

```diff
@@ -0,0 +1,91 @@
+<!-- vim-markdown-toc GFM -->
+
+- [cursorless.nvim](#cursorlessnvim)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+    - [Lazy installation](#lazy-installation)
+    - [Manual installation](#manual-installation)
+  - [Configuration](#configuration)
+    - [neovim configuration](#neovim-configuration)
+    - [Talon configuration](#talon-configuration)
+  - [Frequently asked questions](#frequently-asked-questions)
+    - [nvim does not support Lazy?](#nvim-does-not-support-lazy)
+    - [nvim does not find the `neovim` globally installed package?](#nvim-does-not-find-the-neovim-globally-installed-package)
+  - [Contributors](#contributors)
+
+<!-- vim-markdown-toc -->
+
+# cursorless.nvim
+
+Neovim plugin to support Cursorless
+
+## Prerequisites
+
+- neovim: https://neovim.io/
+- Talon voice: https://talonvoice.com/
+- neovim-talon: https://github.com/hands-free-vim/neovim-talon
+- node/npm: https://nodejs.org/en
+- neovim node package: https://github.com/neovim/node-client (globally installed with `npm`)
+- talon.nvim: https://github.com/hands-free-vim/talon.nvim (optional but recommended)
```

Comment:
mind dropping a link to the PR you have in mind? It seems like it's not optional, because various talon-neovim app activation contexts in https://github.com/hands-free-vim/neovim-talon/blob/main/apps/neovim.py match on `VIM MODE`, which is set by this extension.

Reply:
https://github.com/saidelike/cursorless/pull/3